public/main.js

```diff
@@ -97,12 +98,28 @@ function reloadTodoList() {
             listItem.classList.toggle("completedTodoItem", todo.isComplete);
             todoList.appendChild(listItem);
         });
-        var remainingItems = todos.reduce(function(prev, curr) {
-            return prev + (curr.isComplete ? 0 : 1);
+        var completedItemCount = todos.reduce(function(prev, curr) {
+            return prev + (curr.isComplete ? 1 : 0);
         }, 0);
+        var remainingItemCount = todos.length - completedItemCount;
         countLabel.textContent = (
-            remainingItems.toString() +
-            " item" + (remainingItems === 1 ? "" : "s") + " remaining.");
+            remainingItemCount.toString() +
+            " item" + (remainingItemCount === 1 ? "" : "s") + " remaining.");
+        if (completedItemCount > 0) {
+            var completedItems = todos.filter(function(curr) { return curr.isComplete; });
+            clearCompleteButton.style.visibility = "visible";
+            clearCompleteButton.onclick = function(event) {
+                for (var ii = 0; ii < completedItems.length; ii++) {
```

Comment:
How about you use a more functional style, instead of for loops. `lodash` is a great tool for this: https://lodash.com/.